package.json

```diff
@@ -8,8 +8,9 @@
     "test:all": "run-s compile dist test test-dist test-lib",
     "test-dist": "TEST_SUBJECT=dist node_modules/.bin/mocha --require @babel/register test/.setup.js --recursive test",
     "test-lib": "TEST_SUBJECT=lib node_modules/.bin/mocha --require @babel/register test/.setup.js --recursive test",
-    "compile": "node_modules/.bin/babel src --out-dir lib --copy-files ",
-    "dist": "node_modules/.bin/webpack && npm run build-storybook",
+    "prebuild": "node_modules/.bin/babel src --out-dir lib --copy-files ",
```

Comment:
I love these changes!
shouldn't this be post-build though?
we copy the packages after we create /dist?

Reply:
The order doesn't matter here, dist bundles src to /dist folder. compile transpiles src to /lib and due to --copy-files, it copies the source files also after it's done. I actually don't know why it was decided to do this, lib is the test target when you run test-lib. we should reconsider this in next major version.